__tests__/actions/regions.js

```diff
@@ -0,0 +1,67 @@
+import { Constants } from 'expo';
+import refetch from '../../utils/refetch';
+import * as actions from '../../actions/regions';
+
+jest.mock('../../utils/refetch');
+jest.mock('sentry-expo');
+
+describe('actions/regions', () => {
+  let dispatch;
+  beforeEach(() => {
+    Constants.manifest.extra = Constants.manifest.extra || {};
+    Constants.manifest.extra.mapsUrl = 'mapsUrl';
+
+    dispatch = jest.fn();
+    refetch.mockClear();
+
+    refetch.mockImplementation(async (url) => {
+      if (url === 'mapsUrl/regions/GOOD') {
+        return 'region';
+      }
+
+      if (url === 'mapsUrl/geojson/GOOD') {
+        return 'geojson';
+      }
+
+      throw new Error(url);
+    });
+  });
+
+  describe('request', () => {
+    it('fetches the geojson data', async () => {
+      await actions.request('GOOD')(dispatch);
+
+      expect(refetch).toBeCalledWith(
+        'mapsUrl/geojson/GOOD',
+        expect.objectContaining({ method: 'GET' }),
+      );
+    });
+
+    it('fetches the region metadata', async () => {
```

Comment:
Am I missing something or is this identical to the test above it?

Reply:
They are different routes. I opted to split the geodata and the region data. Geodata will be much larger and we may want to cache them independently.